Userland/Services/WebDriver/Session.cpp

```diff
@@ -158,4 +158,30 @@ ErrorOr<JsonValue, HttpError> Session::get_title()
     return JsonValue(m_browser_connection->get_title());
 }
 
+// POST /session/{session id}/refresh https://w3c.github.io/webdriver/#dfn-refresh
+ErrorOr<void> Session::refresh()
+{
+
```

Comment:
The nittest of picks: Weird empty line here.

Reply:
more love to nitpicks! :^)